FireboltNETSDK/Client/FireboltCommand.cs

```diff
@@ -267,13 +309,8 @@ public void ClearSetList()
             _connection?.SetParamList.Clear();
         }
 
-        /// <summary>
-        /// Not supported. To cancel a command execute it asynchronously with an appropriate cancellation token.
-        /// </summary>
-        /// <exception cref="NotImplementedException">Always throws <see cref="NotImplementedException"/>.</exception>
         public override void Cancel()
         {
-            throw new NotImplementedException();
         }
```

Comment:
Shouldn't we keep the old docstring and error? What's the reasoning here?

Reply:
I've added comment that explains why exception was removed.